x/pool-incentives/types/expected_keepers.go

```diff
@@ -41,3 +47,14 @@ type DistrKeeper interface {
 	SetFeePool(ctx sdk.Context, feePool distrtypes.FeePool)
 	FundCommunityPool(ctx sdk.Context, amount sdk.Coins, sender sdk.AccAddress) error
 }
+type ClKeeper interface {
```

Comment:
nit: let's be verbose and call it as `ConcentratedLiquidityKeeper` please for clarity